docs/contributing.md

```diff
@@ -116,17 +143,42 @@ If you're a contributor and you've forked this repository, you can create previe
 
    - **Yarn >= 2 (berry):**
 
-     Add the following in `.yarnrc.yml`
+     Ensure that the project `.yarnrc.yml` file has the following contents:
```

Comment:
I will de-duplicate these instructions in a follow up PR. I have a few ideas on how to do that already that I've been meaning to try out.

Reply:
I decided to handle this here after all, in https://github.com/MetaMask/core/pull/1481/commits/335daed3c7c5062aa7010afcf1b9a7cde0d66bef